ebu_tt_live/examples/config/retiming_delay_issue_360_example.json

```diff
@@ -0,0 +1,61 @@
+{

```

Comment:
We probably don't want to keep the issue-demonstrating JSON as part of the issue fix, since it is a weird configuration case. If we want to test that the fix is not reintroduced this should be done via unit tests or BDDs. If I don't remember, remind me to remove it before merging.